src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -0,0 +1,1104 @@
+                        id: 'IncoherentRatioRegulatingTerminalError',
+                    })
+                );
+                isFormValid = false;
+            }
+
+            let formatedRatioTapSteps = ratioTapRows.map((row) => {
+                return {
+                    index: row.tap,
+                    r: row.resistance,
+                    x: row.reactance,
+                    g: row.conductance,
+                    b: row.susceptance,
+                    rho: row.ratio,
+                };
+            });
+
+            ratioTap = {
+                loadTapChangingCapabilities:
+                    ratioTapLoadTapChangingCapabilities,
+                regulating: ratioTapRegulating,
+                targetV: targetVoltage,
+                targetDeadband: ratioTapTargetDeadband,
+                regulatingTerminalId:
+                    ratioTapRegulatingTerminal?.equipmentSection?.id,
+                regulatingTerminalType:
+                    ratioTapRegulatingTerminal?.equipmentSection?.type ??
+                    twoWindingsTransformerId ===
+                        ratioTapRegulatingTerminal?.equipmentSection?.id
+                        ? EQUIPMENT_TYPE.TWO_WINDINGS_TRANSFORMER.name
+                        : undefined,
+                regulatingTerminalVlId:
+                    ratioTapRegulatingTerminal?.voltageLevel?.id,
+                tapPosition: ratioTapPosition,
+                lowTapPosition: ratioLowTapPosition,
+                steps: formatedRatioTapSteps,
+            };
+        }
+
+        let phaseTap = {};
+        if (phaseTapChangerEnabled && !phaseTapInputForm.validate()) {
+            isFormValid = false;
+        } else if (phaseTapChangerEnabled && phaseTapInputForm.validate()) {
+            if (
+                phaseTapRegulating &&
+                (!phaseTapRegulatingTerminal?.equipmentSection ||
+                    !phaseTapRegulatingTerminal?.voltageLevel)
+            ) {
+                setCreationError(
+                    intl.formatMessage({
+                        id: 'IncoherentPhaseRegulatingTerminalError',
+                    })
+                );
+                isFormValid = false;
+            }
+
+            let formatedPhaseTapSteps = phaseTapRows.map((row) => {
+                return {
+                    index: row.tap,
+                    r: row.resistance,
+                    x: row.reactance,
+                    g: row.conductance,
+                    b: row.susceptance,
+                    rho: row.ratio,
+                    alpha: row.alpha,
+                };
+            });
+
+            phaseTap = {
+                regulating: phaseTapRegulating,
+                regulationMode: regulationMode,
+                regulationValue:
+                    regulationMode === REGULATION_MODES[2].id
+                        ? flowSetPointRegulatingValue
+                        : regulationMode === REGULATION_MODES[1].id
```

Comment:
See my other comment in constants.js.
I think it's better to call this with REGULATION_MODES.CURRENT_LIMITER.id

Reply:
✓